### PR TITLE
add test artifacts for compliance checking

### PR DESCRIPTION
**Pull request by ckrieger**
Signed-off-by: Christoph Krieger <christoph.krieger@iaas.uni-stuttgart.de>

Added an exemplary compliance rule. Moreover, added a TopologyTemplate that satisfies the rule and a TopologyTemplate that does not satisfy the rule.  

- [ ] Ensure that the commit message is [a good commit message](https://github.com/erlang/otp/wiki/Writing-good-commit-messages)
- [ ] Winery CLI does not report new issues

### CODE REVIEW

File: admin/namespaces/Namespaces.properties [hunk 1]
```diff
@@ -1,29 +1,5 @@
-http\://www.w3.org/2001/XMLSchema = xsd
-http\://www.w3.org/XML/1998/namespace = xmlns
 http\://docs.oasis-open.org/tosca/ns/2011/12 = tosca
 http\://www.opentosca.org/winery/extensions/tosca/2013/02/12 = winery
+http\://www.w3.org/2001/XMLSchema = xsd
+http\://www.w3.org/XML/1998/namespace = xmlns
 http\://www.example.org = ex
-http\://www.opentosca.org/nodetypes = otnt
```
Comment (koppor) [inline admin/namespaces/Namespaces.properties]: You did not other prefix configurations, so please unstage this file from the commit.

File: compliancerules/http%3A%2F%2Fwww.example.org%2Ftosca%2Fcompliancerules/ComplianceRuleWithProperty_w1-wip1/ComplianceRule.tosca [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (koppor) [inline compliancerules/http%3A%2F%2Fwww.example.org%2Ftosca%2Fcompliancerules/ComplianceRuleWithProperty_w1-wip1/ComplianceRule.tosca]: Shouldn't this be a released version? `wip` denotes "work in progress" and that type can be subject of changes.

**Review by koppor — CHANGES_REQUESTED** [review]
I think, all the types should be released, shouldn't they?

File: compliancerules/http%3A%2F%2Fwww.example.org%2Ftosca%2Fcompliancerules/ComplianceRuleWithProperty_w1-wip1/ComplianceRule.tosca (hunk 2 above)
Comment (koppor) [reply compliancerules/http%3A%2F%2Fwww.example.org%2Ftosca%2Fcompliancerules/ComplianceRuleWithProperty_w1-wip1/ComplianceRule.tosca]: @lharzenetter WDYT?

Comment (ckrieger) [reply compliancerules/http%3A%2F%2Fwww.example.org%2Ftosca%2Fcompliancerules/ComplianceRuleWithProperty_w1-wip1/ComplianceRule.tosca]: Yeah sounds reasonable, i didn`t notice the wip

**Pull request closed without merging**